### DISCUSSION TITLE
Issue with image preview when configuring user specific directories

### DISCUSSION THREAD

**Discussion by redherring917b** [prasathmani/tinyfilemanager#827] [Q&A]
Just wondering if this issue of not being able to preview images when configuring user specific directories is expected to be addresses in an upcoming release?  I've seen other discussions from users experiencing the same issue, and some mention of fixes, but I've been unable as of yet to properly sort any of those out.

Nice app by the way!  Just struggling with this one aspect right now.

Perhaps if not a new release that incorporates such a fix, maybe someone would be willing to provide me with their modified config file that sets user specific directories yet still allows image previews to function?

Thanks!

**Comment by msidog2** [comment]
я все эти баги исправил у себя на проекте и сделал для каждого пользователя уникальную папку.. у меня осталась единсвеная проблема это "нет смены имени файлов с кириллицы на латинскую"  метод я один сделал но возникла проблема с загрузкой файлов на сервер.

**Comment by deadlyanony** [comment]
I found a Solution by doing some changes to 2 places. you have to edit the root path and sroot_url path same as shown in the code below. eg. My folder  name is files so I put 'files' at the end of the [DOCUMENT_ROOT]

// Root path for file manager
// use absolute path of directory i.e: '/var/www/folder' or $_SERVER['DOCUMENT_ROOT'].'/folder'
$root_path = $_SERVER['DOCUMENT_ROOT'].'/files';

// Root url for links in file manager.Relative to $http_host. Variants: '', 'path/to/subfolder'
// Will not working if $root_path will be outside of server document root
$root_url = '/files/';
